packages/navie/src/services/context-service.ts

```diff
@@ -36,6 +36,15 @@ export default class ContextService {
       }
 
       const filters = { ...options.buildContextFilters() };
+      if (filters.locations) {
```

Comment:
Can you use the locationContext method instead? 

It already adds the whole file contents.